src/ralph/assets/tests/factories.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import factory
+from factory.django import DjangoModelFactory
+
+from ralph.assets.models.assets import (
+    AssetModel,
+    Environment,
+    Service,
+    ServiceEnvironment
+)
+
+
+class BackOfficeAssetModelFactory(DjangoModelFactory):
+
+    name = factory.Iterator(['3310', 'XD300S', 'Hero 3', 'Computer set'])
+
+    class Meta:
+        model = AssetModel
+
+
+class DataCenterAssetModelFactory(DjangoModelFactory):
+
+    name = factory.Iterator(['DL360', 'DL380p', 'DL380', 'ML10', 'ML10 v21'])
```

Comment:
what will happen if I'll create (in the same test) 6 data center assets (which should reference `DataCenterAssetModelFactory` using `SubFactory`)?